Practice/Practice.DAL/Migrations/201907211855112_ResetPasswordToken.cs

```diff
@@ -0,0 +1,18 @@
+namespace Practice.DAL.Migrations
```

Comment:
Jeśli dodałeś, a następnie usunąłeś coś w migracji tylko lokalnie to wystarczyło cofnąć się do migracji sprzed zmian, a następnie usunąć zbędne migracje.

Reply:
Ok zapamiętam.